twilight-interactions-derive/src/command/model/parse.rs

```diff
@@ -327,16 +327,18 @@ pub fn channel_type(kind: &ChannelType) -> TokenStream {
         ChannelType::GuildVoice => quote!(::twilight_model::channel::ChannelType::GuildVoice),
         ChannelType::Group => quote!(::twilight_model::channel::ChannelType::Group),
         ChannelType::GuildCategory => quote!(::twilight_model::channel::ChannelType::GuildCategory),
-        ChannelType::GuildNews => quote!(::twilight_model::channel::ChannelType::GuildNews),
+        ChannelType::GuildAnnouncement => {
+            quote!(::twilight_model::channel::ChannelType::GuildAnnouncement)
+        }
         ChannelType::GuildStore => quote!(::twilight_model::channel::ChannelType::GuildStore),
-        ChannelType::GuildNewsThread => {
-            quote!(::twilight_model::channel::ChannelType::GuildNewsThread)
+        ChannelType::GuildAnnouncementThread => {
+            quote!(::twilight_model::channel::ChannelType::GuildAnnouncementThread)
```

Comment:
This variant is named `AnnouncementThread`, not `GuildAnnouncementThread` ([docs](https://docs.rs/twilight-model/latest/twilight_model/channel/enum.ChannelType.html)).

```suggestion
        ChannelType::AnnouncementThread => {
            quote!(::twilight_model::channel::ChannelType::AnnouncementThread)
```